pkg/securelaunch/measurement/dmi.go

```diff
@@ -0,0 +1,176 @@
+package measurement
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+
+	"github.com/digitalocean/go-smbios/smbios"
```

Comment:
is there any way to dedup this with pkg/smbios here in u-root? to avoid a growing dependency tree

Reply:
we plan to, but not in this commit..
Ultimately, we would want to re-use dmidecode to give us output based on type passed by user.

Reply:
sgtm